activemq/tests/test_check.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import pytest
+
+
+@pytest.mark.e2e
+def test(dd_agent_check):
+    instance = {}
+    aggregator = dd_agent_check(instance)
+    metrics = [
```

Comment:
nit: you can move this to `common.py` so it can be reused if we add integration tests as well :)

Reply:
We cannot add integration tests with jmx 😢